pandas/io/sql.py

```diff
@@ -1404,6 +1404,15 @@ class SQLiteTable(SQLTable):
     Instead of a table variable just use the Create Table statement.
     """
 
+    def __init__(self, *args, **kwargs):
+        # GH 8341
+        # register an adapter callable for datetime.time object
+        import sqlite3
```

Comment:
I think the idea is to only have this adapter being registered if you actually use pandas with `to_sql`, not if you just import pandas to do other things.

But actually, we should maybe test how this possibly interferes with sqlalchemy. @nbonnotte Can you test that writing/reading time values with sqlalchemy still works after having run this?